integrations/extensions/starter-kits/language-model-conversational-search/discovery-watsonx-actions.json

```diff
@@ -849,6 +849,12 @@
                         "skill_variable": "query_text"
                       },
                       "parameter": "natural_language_query"
+                    },
+                    {
+                      "value": {
+                        "scalar": true
+                      },
+                      "parameter": "passages.per_document"
```

Comment:
It should also be a required parameter in the starter kit, not optional.

Reply:
Umm, I have directly set it here, instead of making it a session variable. 
<img width="929" alt="Screenshot 2023-08-08 at 8 16 58 AM" src="https://github.com/watson-developer-cloud/assistant-toolkit/assets/23107360/2f226ad3-9922-4e3b-8263-af71b524d084">

Reply:
Ah I see.

Reply:
Does both `True` and `true` valid? They should all be consistent.

Reply:
I think both works, true set by expression. I think more accurate one is "select from option" -> which is `True`/ `False`

<img width="930" alt="Screenshot 2023-08-08 at 10 05 37 AM" src="https://github.com/watson-developer-cloud/assistant-toolkit/assets/23107360/a56a6aae-a946-4e29-a37c-831586d0e475">